examples/video_understanding/configs/llms/text_encoder.yml

```
@@ -0,0 +1,3 @@
+name: OpenaiTextEmbeddingV3
+endpoint: ${env| custom_openai_endpoint, https://api.openai.com/v1}
+api_key: ${env| custom_openai_key, openai_api_key}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)